catalog/spatial/csw/spatial-csw-transformer/src/main/java/org/codice/ddf/spatial/ogc/csw/catalog/converter/XstreamPathConverter.java

```diff
@@ -122,15 +123,18 @@ protected boolean doBasicPathsMatch(final Path pathObj1, final Path pathObj2) {
       return true;
     }
 
-    char[] path1 = pathObj1.toString().toCharArray();
-    char[] path2 = pathObj2.toString().toCharArray();
+    String path1Str = pathObj1.toString();
+    String path2Str = pathObj2.toString();
+    char[] path1 = path1Str.toCharArray();
+    char[] path2 = path2Str.toCharArray();
     int i, j;
 
     for (i = 0, j = 0; i < path1.length && j < path2.length; i++, j++) {
       i = countPastTag(path1, i);
       j = countPastTag(path2, j);
       if (i < path1.length && j < path2.length) {
-        if (path1[i] == ATTR_TAG && path2[j] == ATTR_TAG) {
+        if ((path1[i] == ATTR_TAG && path2[j] == ATTR_TAG)
+            || (path1[i] == EVERYTHING_TAG && path2Str.startsWith(path1Str.substring(0, i - 2)))) {
```

Comment:
Should this be matching any element OR any attribute?

Reply:
Also, this only works at the end of the path?
It'd be nice to have this in the Javadoc

Reply:
This is checking for a path match of some sort. If the path matches, then it hits the logic for handling the element or the attribute match.